packages/contracts-bedrock/src/universal/StandardBridge.sol

```diff
@@ -11,17 +11,10 @@ import { CrossDomainMessenger } from "./CrossDomainMessenger.sol";
 import { OptimismMintableERC20 } from "./OptimismMintableERC20.sol";
 import { Initializable } from "@openzeppelin/contracts/proxy/utils/Initializable.sol";
 
-/// @custom:upgradeable
-/// @title StandardBridge
-/// @notice StandardBridge is a base contract for the L1 and L2 standard ERC20 bridges. It handles
-///         the core bridging logic, including escrowing tokens that are native to the local chain
-///         and minting/burning tokens that are native to the remote chain.
-abstract contract StandardBridge is Initializable {
-    using SafeERC20 for IERC20;
-
-    /// @notice The L2 gas limit set when eth is depoisited using the receive() function.
-    uint32 internal constant RECEIVE_DEFAULT_GAS_LIMIT = 200_000;
-
+/// @title StandardBridgeBase
+/// @notice StandardBridgeBase is the root contract that maintains a shared storage layout
+///         between the StandardBridge, ETHStandardBridge, and ERC20StandardBridge.
+abstract contract StandardBridgeBase is Initializable {
```

Comment:
nit: I would prefer every contract to have it's own file. It makes the artifacts easier to target